packages/render-fidelity-tools/src/common.ts

```diff
@@ -24,20 +24,22 @@ export const COMPONENTS_PER_PIXEL: number = 4;
 // @see http://www.progmat.uaem.mx:8080/artVol2Num2/Articulo3Vol2Num2.pdf
 export const MAX_COLOR_DISTANCE: number = 35215;
 
+export const DEVICE_PIXEL_RATIO: number = 2;
```

Comment:
Is this correct on non high DPI displays?

Reply:
Since this is headless in puppeteer, the DPR is not related to a physical display. We only choose to use a non-unity value because handling DPR is bug-prone and this helps with noticing errors.